app/routers/world_clock.py

```diff
@@ -0,0 +1,744 @@
+CITY_NAME_KEY = "name"
+COUNTRY_NAME_KEY_IN_CONTITNENTS = "Country_Name"
+COUNTRY_NAME_KEY_IN_CITIES = "country"
+SUBCOUNTRY_NAME_KEY_IN_CITIES = "subcountry"
+CONTINENT_NAME_KEY = "Continent_Name"
+CONTINENT = 'continent'
+COUNTRY = 'country'
+PLACE = 'place'
+CURRENT_TIME_KEY = 'datetime'
+TIMEZONE_PARTS_MAP = {CONTINENT: 0, PLACE: -1}
+TIME_SCOPE_INDEX = 0
+FEEDBACK_INDEX = 1
+ARB_COUNTRY_TIMEZONE = -1
+PATH_SEPARETOR = '/'
+PARTS_OF_THE_DAY_FEEDBACK = {'Early morning':
+                             [('05:00:00', '07:59:59'), 'Better not'],
+                             'Morning':
+                             [('08:00:00', '10:59:59'), 'OK'],
+                             'Late morning':
+                             [('11:00:00', '11:59:59'), 'OK'],
+                             'Early afternoon':
+                             [('12:00:00', '12:59:59'), 'OK'],
+                             'Afternoon':
+                             [('13:00:00', '15:59:59'), 'OK'],
+                             'Late afternoon':
+                             [('16:00:00', '16:59:59'),
+                              'Can be considered'],
+                             'Early evening':
+                             [('17:00:00', '18:59:59'),
+                              'Can be considered'],
+                             'Evening':
+                             [('19:00:00', '20:59:59'), 'Better not'],
+                             'Night':
+                             [('21:00:00', '23:59:59'), 'Better not'],
+                             'Late night':
+                             [('00:00:00', '04:59:59'),
+                              'Not possible'], }
+
+
+def normalize_continent_name(continent_name: str) -> Optional[str]:
+    """Normalize a given continent name to the continent name
+       that appears in the timezone list.
+
+
+    Args:
+        continent_name (str): The given continent name.
+
+
+    Returns:
+        str: The normalized continent name.
+    """
+    for continent in CONTINENTS:
+        if continent in continent_name:
+            return continent
+    return continent_name
+
+
+def get_continent(country_name: str) -> Optional[str]:
+    """Get the continent of a given country.
+
+
+    Args:
+        country_name (str): The given country name.
+
+
+    Returns:
+        str: The suitable continent name.
+    """
+    details = requests.get(COUNTRY_TO_CONTINENT_DATA_SET).json()
+    for country_element in details:
+        if country_name.title() in \
+                country_element[COUNTRY_NAME_KEY_IN_CONTITNENTS]:
+            return normalize_continent_name(
+                country_element[CONTINENT_NAME_KEY])
+
+
+def get_country(city_name: str) -> Optional[str]:
+    """Get the country of a city.
+
+
+    Args:
+        city_name (str): The given city name.
+
+
+    Returns:
+        str: The suitable country name.
+    """
+    details = requests.get(CITY_TO_COUNTRY_DATA_SET).json()
```

Comment:
We shouldn't use `requests` on the server side. Why we need to send a request to the server? Prefer using JS to find it

Reply:
Actually, I do not quite sure how to integrate js here... Isn't it better to save the data as a file and thus it will not be necessary to send a request to the server?